translations/README.md

```diff
@@ -0,0 +1,19 @@
+## Translation guidelines
+
+### General guidelines
+
+When possible, please use the following guidelines when translating field names:
+
+- Entity type names should **not** be included in the label, when it is redundant
+  - _Example_: `First name` instead of `User first name`
+- Field names should have only their first word capitalized, per FOLIO UI guidelines
+  - _Example_: `First name`, `Effective call number`
+- Acronyms such as POL, ID, HRID, etc., should always be fully capitalized
+- As long as it is not ambiguous, use 'ID' instead of 'UUID'
```

Comment:
The only place I'm unsure of this is "Instance UUID" — it seems the inventory app uses this explicitly, likely to clarify against HRID

Reply:
Didn't Magda say that the FOLIO convention is UUID instead of ID?

Reply:
This one is tough. It bothers me that most fields are just ID but then we have "Instance UUID". Since firebird specifically requested that it be "Instance UUID", we'd either need to change them all IDs to UUID, or just let this be inconsistent.

Reply:
I thing it's better to make them as UUID. So that things remains consistent across the board